src/test/scala/latis/service/hapi/ToHapiTimeSpec.scala

```diff
@@ -95,7 +95,7 @@ class ToHapiTimeSpec extends AnyFlatSpec{
     val disp = Scalar.fromMetadata(Metadata("id"->"displacement", "type"->"int", "units"->"meters")).value
     val model = Function.from(time, disp).value
     val data = new SeqFunction(Seq(
-      Sample(time.convertValue("3661.001").toSeq, RangeData(1)),
+      Sample(time.parseValue("3661.001").toSeq, RangeData(1)),
```

Comment:
Would this work?: `DomainData(3661.001)`

Reply:
Ya, it does. I had tried doing that but without removing the quotes and it didn't, but without the quotes it does, except for the final `handle time in milliseconds since 1970`, because `946688461001` is too big to be an int. Not sure how we want to approach this group of tests given that....

Reply:
Nevermind, it works if I stick an L at the end of the number to force it to be a long.

Reply:
See my comment elsewhere about using `.0` or `D` since the time type is `double` in that case.